### PR TITLE
Update torch to 1.13.1+cu116

### LINKED ISSUE #644: [BUG]: pytorch CVE
### Version

23.01

### Which installation method(s) does this occur on?

Docker

### Describe the bug.

Current version of torch 1.12.0+cu113 has a Critical CVE

https://github.com/advisories/GHSA-47fc-vmwq-366v

Recommend update to 1.13.1

### Minimum reproducible example

```shell
Anchore scanning
```

### Relevant log output

_No response_

### Full env printout

_No response_

### Other/Misc.

_No response_

### Code of Conduct

- [X] I agree to follow Morpheus' Code of Conduct
- [X] I have searched the [open bugs](https://github.com/nv-morpheus/Morpheus/issues?q=is%3Aopen+is%3Aissue+label%3Abug) and have found no duplicates for this bug report

### PR DESCRIPTION
**Pull request by pdmack**
Fixes #644

### CODE REVIEW

**Comment by efajardo-nv** [comment]
@pdmack versions should also be updated here:
https://github.com/nv-morpheus/Morpheus/blob/branch-23.01/setup.py#L52
https://github.com/nv-morpheus/Morpheus/blob/branch-23.01/models/training-tuning-scripts/phishing-models/requirements.txt#L6
https://github.com/nv-morpheus/Morpheus/blob/branch-23.01/models/training-tuning-scripts/root-cause-models/requirements.txt#L6
https://github.com/nv-morpheus/Morpheus/blob/branch-23.01/models/validation-inference-scripts/dfp-models/requirements.txt#L9
https://github.com/nv-morpheus/Morpheus/blob/branch-23.01/models/validation-inference-scripts/phishing-models/requirements.txt#L5
https://github.com/nv-morpheus/Morpheus/blob/branch-23.01/models/validation-inference-scripts/root-cause-models/requirements.txt#L5

**Comment by pdmack** [comment]
@efajardo-nv ack

**Comment by pdmack** [comment]
@efajardo-nv PTAL

**Review by efajardo-nv — APPROVED** [review]
LGTM

**Review by mdemoret-nv — APPROVED** [review]
